src/pages/createConfig/step2AddConfig/step2AddConfig.tsx

```diff
@@ -69,7 +69,7 @@ export const Step2AddConfig: React.FC = () => {
     try {
       const json = JSON.parse(stringValue);
       const { isValid, errors } = await validateJson(schema, json);
```

Comment:
```suggestion
      const { isValid, errors } = await validateJson(schema, JSON.parse(stringValue));
```

Reply:
1. this is out of scope of the PR.
2. i think that readability is more important.

Reply:
OK